test/request.ts

```diff
@@ -456,6 +456,53 @@ describe('Request', () => {
           .emit('reading');
       });
 
+      it('should pass `wrapNumbersOprions` to formatArray', done => {
```

Comment:
I think these tests should be rewritten to assure all the asserts are called. If necessary, break them apart into multiple tests for each value type.

Reply:
Split all these tests into single test per each value.